### ISSUE TITLE
ImportError: libhdf5.so.10: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by johannesring** [conda-forge/fenics-feedstock#47]
After adding HDF5 the other day everything seemed to be working fine, however, now I get this error when importing dolfin:

``` console
$ python -c "import dolfin"
Traceback (most recent call last):
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/site-packages/dolfin/cpp/common.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 648, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 560, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
ImportError: libhdf5.so.10: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/site-packages/dolfin/__init__.py", line 17, in <module>
    from . import cpp
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/site-packages/dolfin/cpp/__init__.py", line 43, in <module>
    exec("from . import %s" % module_name)
  File "<string>", line 1, in <module>
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/site-packages/dolfin/cpp/common.py", line 21, in <module>
    _common = swig_import_helper()
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/site-packages/dolfin/cpp/common.py", line 20, in swig_import_helper
    return importlib.import_module('_common')
  File "/home/johannr/miniconda2/envs/fenicsproject-py36/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_common'
```
I see that a new version of HDF5 was uploaded yesterday (in PR [#73](https://github.com/conda-forge/hdf5-feedstock/pull/73)) and that could be the reason for this error.

**Comment by johannesring** [comment]
I assume that this will be fixed if we rebuild against the new HDF5.

**Comment by johannesring** [comment]
We should probably pin a specific version of HDF5 in build/run requirements to avoid problems like [this](https://groups.google.com/forum/#!topic/fenics-support/2j9_XwGwixM).

**Comment by johannesring** [comment]
Fixed now in PR #48.
